modules/configuring-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+//
+//* Installing-vsphere-installer-provisioned-customizations.adoc [IPI]
+//* installing-vsphere-installer-provisioned-network-customizations.adoc [IPI]
+//* installing-vsphere.adoc [UPI]
+//* installing-vsphere-network-customizations.adoc [UPI]
+//* installing-restricted-networks-installer-provisioned-vsphere.adoc [IPI]
+//* installing-restricted-networks-vsphere.adoc [IPI]
+
+:_content-type: PROCEDURE
+[id="configuring-vsphere-regions-zones_{context}"]
+= Configuring regions and zones for a VMware vCenter
+You can modify the default installation configuration file, so that you can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter. 
+
+[NOTE]
+====
+The default `install-config.yaml` file configuration from the previous release of {product-title} is deprecated. You can continue to use the deprecated default configuration, but the `openshift-installer` will prompt you with a warning message that indicates the use of deprecated fields in the configuration file. 
+====
+
+[IMPORTANT]
+====
+The example uses the `govc` command. The `govc` command is an open-source command available from VMware; it is not available from Red Hat. The Red Hat support team does not maintain the `govc` command. Instructions for downloading and installing `govc` are found on the VMware documentation website
+====
+
+.Prerequisites
+* You have an existing `install-config.yaml` installation configuration file.
++
+[IMPORTANT]
+====
+You must specify at least one failure domain for your {product-title} cluster, so that you can provision datacenter objects for your VMware vCenter server. Consider specifying multiple failure domains if you need to provision virtual machine nodes in different datacenters, clusters, datastores, and so on.
```

Comment:
@dfitzmau Style guide "wants" the words in datacenter to be separate, as in "data center". But I believe, sir, in this context, you are talking about vSphere "datacenter", which is a meaningful noun unto itself, right?

Reply:
@dfitzmau I would not normally just comment like this, but I think I wanted to leave it here for the benefit of any further reviewers down the line.

Reply:
Yes, you are correct in that "datacenter" is OK in the context of vSphere and VMC.